src/Entity/OgMembership.php

```diff
@@ -204,7 +212,11 @@ public function setState(string $state): OgMembershipInterface {
    * {@inheritdoc}
    */
   public function getState(): string {
-    return $this->get('state')->value;
+    $state = $this->values['state'][LanguageInterface::LANGCODE_DEFAULT] ?? $this->get('state')->value;
```

Comment:
To be a bit more specific and helpful, what I noticed from my phone is for example this, which inverts the pattern and looks for $this->values first. But if you changed the state then $this->values might not be up to date anymore, it will only be synced back when an entity is serialized/saved.

Not sure why this isn't using your method?

Reply:
Ohh thanks for pointing this out, this is still using an approach I was trying out earlier in my discovery. This needs to use the method too, and this means we don't have adequate test coverage for this method.

Reply:
Whoops I missed it too.